README.md

```diff
@@ -3,9 +3,107 @@ A Clojure library for interacting with the github developer API.
 
 Note: while this library is being used for several production use cases, we're still ironing out the APIs, so they are subject to change.
 
-## tests
+## Httpkit client
 
-To run tests:
+Useful for non service environments (e.g. cicd pipelines, lambdas, etc).
```

Comment:
I think this could be removed (this was only relevant inside Nubank and the idea is to eventually deprecate the internal component we use for services).
```suggestion
```